tests/bluetooth/tester/src/gap.c

```diff
@@ -130,6 +131,7 @@ static void supported_commands(uint8_t *data, uint16_t len)
 	tester_set_bit(cmds, GAP_PASSKEY_CONFIRM);
 	tester_set_bit(cmds, GAP_CONN_PARAM_UPDATE);
 	tester_set_bit(cmds, GAP_SET_MITM);
+        tester_set_bit(cmds, GAP_OOB_LEGACY_SET_DATA);
```

Comment:
please use tabs and not spaces to conform with Zephyr's coding style

Reply:
Will do that although I feel strange to use tabs